rollup/proposer-tool-config.json

```diff
@@ -2,8 +2,8 @@
     "l2_config": {
       "endpoint": "https://rpc.scroll.io",
       "chunk_proposer_config": {
-        "max_block_num_per_chunk": 100,
-        "max_l2_gas_per_chunk": 20000000,
+        "max_block_num_per_chunk": 1000,
```

Comment:
Why these updates to the config?

Reply:
This is changed to test the previous experimental dynamic encoding of `codecv8`, to remove other limits, so setting large values in them.

Reply:
Reverted in commit [d274123](https://github.com/scroll-tech/scroll/pull/1681/commits/d274123c84bd57502028ebeabeff81d97723bb34).